app/src/util/form-schema/manual-input-hierarchy.json

```diff
@@ -1467,10 +1522,25 @@
               }
             ]
           }
+        ],
+        "suggestedActivities": [
+          {
+            "id": "electricity-consumption-on-road-transport-activity",
+            "on-road-transport-vehicle-type": "vehicle-type-all"
+          },
+          {
+            "id": "electricity-consumption-on-road-transport-activity",
+            "on-road-transport-vehicle-type": "vehicle-type-passenger-vehicles"
+          },
+          {
+            "id": "electricity-consumption-on-road-transport-activity",
+            "on-road-transport-vehicle-type": "vehicle-type-public-transport-vehicles"
+          }
         ]
       }
     ],
     "directMeasure": {
+      "id": "on-road-transport-direct-measure-energy-consumption-methodology",
       "extra-fields": [
         {
           "id": "on-road-transport-vehicle-type",
```

Comment:
in "on-road-transport-fuel-type" is missing:
- the option: "fuel-type-all"
- "exclusive": "fuel-type-all"